example/pubspec.yaml

```diff
@@ -27,6 +27,7 @@ environment:
 # the latest version available on pub.dev. To see which dependencies have newer
 # versions available, run `flutter pub outdated`.
 dependencies:
+  cupertino_icons: ^1.0.2
```

Comment:
do we need cupertino_icons ?

Reply:
It's in the example project so I'd say that it doesn't matter.